Assets/Scripts/Models/Character/Character.cs

```diff
@@ -425,7 +429,7 @@ public string GetDescription()
 
     public IEnumerable<string> GetAdditionalInfo()
     {
-        yield return string.Format("HitPoints: 100/100");
+        yield return string.Format("HitPoints: " + currentHP + "/" + maxHP);
```

Comment:
this is a weird use of string.Format.. usually you go with 
`string.Format("HitPoints: {0}/{1} ", currentHP, maxHP);`
to prevent creation of small strings that are GCed later.